includes/data/mutation/class-cart-mutation.php

```diff
@@ -117,12 +117,12 @@ private static function prepare_attributes( $product_id, array $variation_data =
 	 * @throws UserError Cart item not found message.
 	 */
 	public static function retrieve_cart_items( $input, $context, $info, $mutation = '' ) {
+		$items = [];
 		if ( ! empty( $input['all'] ) && $input['all'] ) {
 			$items = array_values( \WC()->cart->get_cart() );
 		}
-
-		if ( ! empty( $input['keys'] ) && ! isset( $items ) ) {
-			$items = [];
+		
+		if ( ! empty( $input['keys'] ) && ! empty( $items ) ) {
```

Comment:
This change means that if `WC()->cart->get_cart()` returns empty, the foreach loop will no longer run.

If that's not intentional, then initialize `$items` to `null`, and then change this to `&& null !== $items`